sdccc/src/test/java/com/draeger/medical/sdccc/messages/TestMessageStorage.java

```diff
@@ -1449,6 +1453,7 @@ public void testMessageStorageFlushNotInDeadlock(@TempDir final File dir) throws
                     2000,
                     ResponseTypes.Result.RESULT_SUCCESS,
                     "setMetricStatus",
+                    "expectedMethod",
```

Comment:
the new response value should be added after the result and before the "setMetricStatus" (which is the expectedMethod already. Then rename to e.g. "expectedResponse"
But the test case run successful, so it is optional.